src/lang/english.txt

```diff
@@ -5084,6 +5084,20 @@ STR_ERROR_CAN_T_COPY_ORDER_LIST                                 :{WHITE}Can't co
 STR_ERROR_TOO_FAR_FROM_PREVIOUS_DESTINATION                     :{WHITE}... too far from previous destination
 STR_ERROR_AIRCRAFT_NOT_ENOUGH_RANGE                             :{WHITE}... aircraft has not enough range
 
+# Extra messages which go on the third line of errors,
+# explaining why orders failed
+STR_ERROR_NO_RAIL_STATION                                       :{WHITE}There is no railway station
+STR_ERROR_NO_BUS_STATION                                        :{WHITE}There is no bus station
+STR_ERROR_NO_TRUCK_STATION                                      :{WHITE}There is no lorry station
+STR_ERROR_NO_DOCK                                               :{WHITE}There is no dock
+STR_ERROR_NO_AIRPORT                                            :{WHITE}There is no airport/heliport
+STR_ERROR_NO_STOP_COMPATIBLE_ROAD_TYPE                          :{WHITE}There are no stops with a compatible road/tram type
+STR_ERROR_NO_STOP_ARTIC_VEH                                     :{WHITE}There are no stops which are suitable for articulated road vehicles.{}Articulated road vehicles require a drive-through stop, not a bay stop
```

Comment:
Is "bay stop" the usual term? I'm not aware of it used anywhere else

Reply:
Hmm, the tooltip for freight road stops says "Lorry Loading Bay," as does the tooltip for choosing the orientation. They're also called that on the smallmap. Bus stations are just called "bus station."

If you have a better suggestion I'd welcome it. A "bay stop" makes sense to me and I'm not sure if we call it anything else elsewhere.